app/overnight/revert-expired-insurance.js

```diff
@@ -0,0 +1,89 @@
+const sequelize = require('../config/db')
+const { Op } = require('sequelize')
+const { statuses, breachReasons } = require('../constants/statuses')
+const { dbFindAll, dbFindOne } = require('../lib/db-functions')
+const { updateStatusOnly } = require('../repos/status')
+const { removeBreachReasonFromDog } = require('../repos/breaches')
+
+const findExpired = async (currentStatus, t) => {
+  return await dbFindAll(sequelize.models.registration, {
+    where: {
+      certificate_issued: {
+        [Op.ne]: null
+      },
+      '$dog.status.status$': currentStatus,
+      '$exemption_order.exemption_order$': '2023'
+    },
+    include: [
+      {
+        model: sequelize.models.exemption_order,
+        as: 'exemption_order'
+      },
+      {
+        model: sequelize.models.dog,
+        as: 'dog',
+        include: [
+          {
+            model: sequelize.models.status,
+            as: 'status'
+          },
+          {
+            model: sequelize.models.insurance,
+            as: 'insurance'
+          },
+          {
+            model: sequelize.models.dog_breach,
+            as: 'dog_breaches',
+            include: [
+              {
+                model: sequelize.models.breach_category,
+                as: 'breach_category'
+              }
+            ]
+          }
+        ]
+      }
```

Comment:
Not essential, just an idea, but looks like you pull out the relationships into a shared const